iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/persistence/ModelInfo.java

```diff
@@ -80,6 +80,7 @@ public class ModelInfo implements SnapshotProcessor {
     builtInForecastModel.add("_NaiveForecaster");
     builtInForecastModel.add("_STLForecaster");
     builtInForecastModel.add("_ExponentialSmoothing");
+    builtInForecastModel.add("_sundial");
```

Comment:
I think we should also add the limitation of input line number for model. Just like https://github.com/apache/iotdb/pull/15584/commits/df052b59044bd5f7a365ed77b3eaf4e58940659a